examples/hello-world-pyprojecttoml/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["setuptools", "wheel", "setuptools-rust"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "hello-world"
+version = "1.0"
+
+[project.scripts]  # Python entry-point wrapper to be installed in `$venv/bin`
+sum-cli = "hello_world.sum_cli:main"
+
+[tool.setuptools.packages]  # pure Python packages/modules
+find = { where = ["python"] }
+
+[[tool.setuptools-rust.ext-modules]]
```

Comment:
Similar to `[[tool.setuptools-rust.bin]]` suggestion below, should this be `[[tool.setuptools-rust.lib]]`? I'm less sure about this one, mostly because `Cargo.toml` only allows one `[lib]` but we allow multiple here.

Reply:
I don't have strong feelings about this one either.
However I do recognise that from the Python user perspective (e.g. a Python dev that finds himself in the situation where he has/wants to include Rust binaries in a package - such as wrapping Rust libraries or improving performance), `ext-modules` may be a more descriptive term.

On top of that, as you mentioned, Rust only seem to accept one `[lib]` (I don't have much Rust experience, so that came as a surprise for me yesterday - but I suppose you can side-step that by having multiple `Cargo.toml` files).

Once you have a final view on what you would like the terms to be, please let me know and I will implement a new commit.

Reply:
I think most the Python `pyproject.toml` metadata uses plural, so let's keep `ext-modules`.